webapp/channels/src/components/profile_popover/profile_popover_override_disclaimer.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+type Props = {
+    username: string;
+    haveOverrideProp: boolean;
+}
+const ProfilePopoverOverrideDisclaimer = ({
+    username,
+    haveOverrideProp,
+}: Props) => {
+    if (!haveOverrideProp) {
+        return null;
+    }
+
+    return (
+        <div
+            data-toggle='tooltip'
+            key='user-popover-settings'
```

Comment:
These components don't actually need to have `key`  props any more since they're not being pushed into an array of components